libpromises/parser_state.h

```diff
@@ -59,6 +59,7 @@ typedef struct
     bool references_body;
 
     char *promiser;
+    Rval promiser_attribute;
```

Comment:
There are no comments on anything else there!!! Why start now?!?! :)

The name can be adjusted with a global search and replace, but I'm not eager to change the implementation, see my other comment... too much risk IMO.